pagopa/io-app
# chore(IT Wallet): [SIW-940] Add new wallet section screen with empty state

## Short description
This PR adds the new wallet screen inside the home tab navigator. The screen includes its empty state, which displays a message with a CTA.

## List of changes proposed in this pull request
- Added `WalletHomeScreen` and `WalletEmptyScreenContent` components
- Added required locale keys

## How to test
Enable the new wallet UI with the `New wallet section` toggle in the profile section. You should be able to see the new wallet UI as seen in the preview video below.

## Preview

https://github.com/pagopa/io-app/assets/6160324/f966af2d-2db8-4a5f-b574-2cc914b18aef
